tokio-util/src/lib.rs

```diff
@@ -59,3 +59,5 @@ pub mod either;
 pub use bytes;
 
 mod util;
+
+pub mod future;
```

Comment:
Since exposing a new module at the crate root is a significant change, I'd like to hear @Darksonn 's opinion. Does this change make sense for you?